AUTHORS

```diff
@@ -132,15 +140,22 @@ Colin Rice <colin@daedrum.net>
 Colin Walters <walters@verbum.org>
 Cory Forsyth <cory.forsyth@gmail.com>
 cressie176 <github@stephen-cresswell.net>
+Cristian Staretu <cristian.staretu@gmail.com>
 Cruceru Calin-Cristian <crucerucalincristian@gmail.com>
 Daan van Berkel <daan.v.berkel.1980@gmail.com>
+daehyeok mun <daehyeok@daehyeokui-MacBook-Air.local>
+Daehyeok Mun <daehyeok@gmail.com>
 Daehyeok.Mun <daehyeok@gmail.com>
```

Comment:
lol `daehyeokui-MacBook-Air.local` seems legit. I'll flatten these 3 in `.mailmap`